WordPress/gutenberg
# Migrate customize widgets feature preferences to interface package

## Description
Partly addresses #31965. Related to #31965.

Migrates customize widgets feature preferences over to the interface package.

## How has this been tested?
1. Checkout `trunk` and open the customize widgets editor.
2. Set 'Top toolbar' and 'Contain cursor inside block' to active.
3. Checkout this branch, rebuild and open up the customize widgets editor
4. The welcome guide should not show and the top toolbar and contain cursor settings should be retained from step 2.
5. Clear browser storage (or just the `WP_DATA_USER_` values) and reload
6. Open up the customize widgets editor, the welcome guide should be shown
7. Try toggling various settings to ensure they work.

## Types of changes


New feature (non-breaking change which adds functionality)


## Checklist:
